VAMobile/src/screens/HealthScreen/TravelPay/TravelPayClaims/TravelPayClaimsFilter.tsx

```diff
@@ -0,0 +1,180 @@
+  }
+
+  const getPickerOptions = (): Array<TravelClaimsDatePickerOption> => {
+    const todaysDate = DateTime.local()
+    const threeMonthsEarlier = todaysDate.minus({ months: 3 })
+
+    return [
+      {
+        label: t('travelPay.statusList.pastThreeMonths'),
+        value: t('travelPay.statusList.pastThreeMonths'),
+        a11yLabel: t('travelPay.statusList.pastThreeMonths'),
+        dates: { startDate: threeMonthsEarlier.startOf('day'), endDate: todaysDate.endOf('day') },
+      },
+    ]
+  }
+
+  const pickerOptions = getPickerOptions()
+  const [datePickerOption, setDatePickerOption] = useState(pickerOptions[0])
+
+  const setValuesOnPickerSelect = (selectValue: string): void => {
+    const curSelectedRange = pickerOptions.find((el) => el.value === selectValue)
+    if (curSelectedRange) {
+      setDatePickerOption(curSelectedRange)
+    }
+  }
+
+  // TODO- add modal props
+  const modalProps: RadioGroupModalProps = {
+    groups: [
+      {
+        items: [
+          { optionLabelKey: 'All', value: 'all' },
+          { optionLabelKey: 'Paid', value: 'paid' },
+          { optionLabelKey: 'Denied', value: 'denied' },
+        ],
+        onSetOption: (newFilter: string) => {
+          setSelectedFilter(newFilter)
+        },
+        selectedValue: selectedFilter,
+        title: 'Filter by',
+      },
+      {
+        items: [
+          { optionLabelKey: 'Most recent', value: 'recent' },
+          { optionLabelKey: 'Oldest', value: 'oldest' },
+        ],
+        onSetOption: (newSortBy: string) => {
+          setSelectedSortBy(newSortBy)
+        },
+        selectedValue: selectedSortBy,
+        title: 'Sort by',
+      },
+    ],
+    buttonText: t('filterAndSort'),
+    buttonA11yLabel: t('filterAndSort'),
+    buttonTestID: 'openFilterAndSortTestID',
+    headerText: t('filterAndSort'),
+    onApply: () => {
+      // TODO- add apply logic
+      console.log('Apply filters:', selectedFilter, selectedSortBy)
+    },
+    onCancel: () => {
+      // TODO- add cancel logic
+      console.log('Cancel filters')
+    },
+    testID: 'TravelPayFilterModalTestID',
+  }
+
+  const filterContainerProps: BoxProps = {
+    display: 'flex',
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    mx: theme.dimensions.gutter,
+    mb: theme.dimensions.standardMarginBetween,
+  }
+
+  const onClearFiltersPress = () => {
+    // TODO- add clear filters functionality
+    console.log('Clear filters pressed')
+  }
+
+  const filterModal = () => {
+    return (
+      <Box {...filterContainerProps}>
+        <Box mr={8} mb={10}>
+          <RadioGroupModal {...modalProps} />
+        </Box>
+        <TextView
+          onPress={onClearFiltersPress}
+          accessibilityRole="button"
+          accessibilityLabel={t('travelPay.statusList.clearFilters')}
+          accessibilityHint="Clears all applied filters and sorting options"
```

Comment:
use a translations function here
```suggestion
          accessibilityHint={t('travelPay.statusList.clearFilters.a11yHint')}
```